ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/monitor-telemetry-postcard-policy.j2

```diff
@@ -0,0 +1,66 @@
+{#
+ Copyright (c) 2023-2024 Arista Networks, Inc.
+ Use of this source code is governed by the Apache License 2.0
+ that can be found in the LICENSE file.
+#}
+{# eos - monitor telemetry postcard policy #}
+{% if monitor_telemetry_postcard_policy is arista.avd.defined %}
+!
+monitor telemetry postcard policy
+{%     if monitor_telemetry_postcard_policy.disabled is arista.avd.defined(false) %}
+   no disabled
+{%     endif %}
+{%     if monitor_telemetry_postcard_policy.ingress.sample.rate is arista.avd.defined %}
+   ingress sample rate {{ monitor_telemetry_postcard_policy.ingress.sample.rate }}
+{%     elif monitor_telemetry_postcard_policy.ingress.sample.tcp_udp_checksum.value is arista.avd.defined and monitor_telemetry_postcard_policy.ingress.sample.tcp_udp_checksum.mask is arista.avd.defined %}
+   ingress sample tcp-udp-checksum value {{ monitor_telemetry_postcard_policy.ingress.sample.tcp_udp_checksum.value }} mask {{ monitor_telemetry_postcard_policy.ingress.sample.tcp_udp_checksum.mask }}
+{%     endif %}
+{%     if monitor_telemetry_postcard_policy.marker_vxlan is arista.avd.defined(true) %}
+   marker vxlan
+{%     elif monitor_telemetry_postcard_policy.vxlan_marking_bit is arista.avd.defined %}
+   marker vxlan header word 0 bit {{ monitor_telemetry_postcard_policy.vxlan_marking_bit }}
+{%     endif %}
+{%     if monitor_telemetry_postcard_policy.ingress.collection.source is arista.avd.defined and monitor_telemetry_postcard_policy.ingress.collection.destination is arista.avd.defined %}
+{%         set ingress_cli = "ingress collection gre source " ~ monitor_telemetry_postcard_policy.ingress.collection.source ~ " destination " ~ monitor_telemetry_postcard_policy.ingress.collection.destination %}
+{%         if monitor_telemetry_postcard_policy.ingress.collection.version is arista.avd.defined(2) %}
+{%             set ingress_cli = ingress_cli ~ " version 2" %}
+{%         endif %}
```

Comment:
I appreciate the logic here, but we should not maintain EOS defaults in eos_cli_config_gen, so if the user gives us version 1 we should print it. (I know EOS will hide it).

Reply:
Fixed.